inference-contract/docs/notebooks/templates/token-issuer.py

```diff
@@ -0,0 +1,232 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.16.1
+#   kernelspec:
+#     display_name: Python 3 (ipykernel)
+#     language: python
+#     name: python3
+# ---
+
+# %% [markdown]
+# # Token Issuer Notebook
+# This section enables customization of the token that will be minted. Edit the variables in the section below as necessary. 
+#
+# identity : the identity of the token creator
+# service_host : the host for the eservice where tokens will be minted, this will use the default service group
+# token_class : the name of tokens that will be generated, this is only used to simplify local access (e.g. context file name)
+# token_description : a description of the asset associated with the minted tokens
+# token_metadata : additional information about the token
+# count : the number of tokens to mint for the asset
+
+# Note that the notebook assumes that there is a key file for the identity of the form: `${keys}/${identity}_private.pem`.
+
+
+# %% tags=["parameters"]
+token_owner = 'user1'
+token_class = 'mytoken'
+token_description = 'this is my token'
+token_metadata = 'created by {}'.format(token_owner)
+count = 1
+service_host = 'localhost'
+instance_identifier = ''
+
+# %% [markdown]
+# <hr style="border:2px solid gray">
+#
+# ## Initialize
+
+# %%
+import os
+import pdo.contracts.jupyter as pc_jupyter
+import IPython.display as ip_display
+
+pc_jupyter.load_ipython_extension(get_ipython())
+
+
+# %% [markdown]
+# ### Initialize the PDO Environment
+#
+# Initialize the PDO environment. This assumes that a functional PDO configuration
+# is in place and that the PDO virtual environment has been activated. In particular, 
+# ensure that the groups file and eservice database have been configured correctly. 
+# If you do not have a service groups configuration, you can create it for a single
+# service host by running the following:
+
+# %%
+# %%skip True
```

Comment:
> Why is skip default True here? Seems to contract text from above? In any case above we should say when to skip or not and that one correspondingly has to change the code block? Given that we have creation conditional on existence of toml file and so re-eval block is safe, why even bother with %%skip at alll?

this only need to be run once per "session". note it will be changing completely soon.